Owasp-ZAP-jwt-addon.yml

```diff
@@ -0,0 +1,14 @@
+---
```

Comment:
@preetkaran20 This needs to be inside _data/projects folder. please relocate this file.